cdap-docs/integrations/source/cask-market.rst

```diff
@@ -184,7 +190,7 @@ and a list of actions required to install the package:
             {
               "name": "steps",
               "value": [
-                "Download the ZIP file from MySQL at https://dev.mysql.com/downloads/file/?id=462850",
+                "Download the ZIP file from MySQL at \https://dev.mysql.com/downloads/file/?id=462850",
```

Comment:
If there is no backslash, Sphinx turns this into an active HTML link. That might be OK, but I have tried to not have active links in the code samples. There are arguments pro and con: pro is that a link checker can see it and actually test if it works. Cons is that it looks distracting. It's actually not an active link in the UI, which is weird IMO.